catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -144,13 +145,24 @@ public UpdateRequest process(UpdateRequest input)
             }
         }
 
+        Attribute validationErrors = metacard.getAttribute(Validation.VALIDATION_ERRORS);
+        Attribute validationWarnings = metacard.getAttribute(Validation.VALIDATION_WARNINGS);
+
+        if (validationErrors != null) {
+            errors.addAll(validationErrors.getValues());
```

Comment:
I may be misunderstanding you. We have configurations that control whether warnings are included in search results or not, and what user attribute is required to view them if they are filtered.

The presence of any warning or error definitionally renders a metacard 'invalid', but there are many configurations that control what happens depending on the validator involved, the severity of the marking, and the permissions of the user.